sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -393,16 +393,19 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
    * also fetch even Deferred messages (but not Deadlettered message).
    *
    * @param messageCount - The number of messages to retrieve. Default value `1`.
+   * @param omitMessageBody - Whether to omit message body when peeking
```

Comment:
Should it mention the default value for it?
```suggestion
   * @param omitMessageBody - Whether to omit message body when peeking. Default is false.
```

Reply:
Fixed